test/views/houseConsignment/index/documents/AdditionalInformationViewSpec.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.houseConsignment.index.documents
+
+import forms.Constants.maxDocumentsAdditionalInfoLength
+import forms.DocumentsAdditionalInformationFormProvider
+import generators.Generators
+import models.NormalMode
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalacheck.Gen
+import play.api.data.Form
+import play.twirl.api.HtmlFormat
+import viewModels.houseConsignment.index.documents.AdditionalInformationViewModel
+import views.behaviours.CharacterCountViewBehaviours
+import views.html.houseConsignment.index.documents.AdditionalInformationView
+
+class AdditionalInformationViewSpec extends CharacterCountViewBehaviours with Generators {
+
+  private val viewModel: AdditionalInformationViewModel =
+    arbitrary[AdditionalInformationViewModel].sample.value
+
+  override def form: Form[String] = new DocumentsAdditionalInformationFormProvider()(viewModel.requiredError)
+
+  override def applyView(form: Form[String]): HtmlFormat.Appendable =
+    injector
+      .instanceOf[AdditionalInformationView]
+      .apply(form, mrn, arrivalId, NormalMode, viewModel, houseConsignmentIndex, documentIndex)(fakeRequest, messages)
+
+  override val prefix: String = Gen
+    .oneOf(
+      "houseConsignment.index.documents.additionalInformation.NormalMode",
+      "houseConsignment.index.documents.additionalInformation.CheckMode"
+    )
+    .sample
+    .value
```

Comment:
For the prefix is can be any strings and will still pass, fine to leave as gen one of those that you have in, but would you consider in the apply view  to pass in  mode rather than Normal Mode and have a 
  private val mode: Mode          = Gen.oneOf(NormalMode, CheckMode).sample.value
  This would check the title etc in Normal Mode and CheckMode